.vale.ini

```diff
@@ -0,0 +1,6 @@
+StylesPath = website/src/writing-guide/styles
+Vocab = website/src/writing-guide/vocab
```

Comment:
This didn't work for me locally after installing `brew install vale`
I suspect it is because the action docker image uses an older version of `vale`.
The docs for `Vocab` specify a different structure than in this PR

Reply:
I haven't evaluated the local version at all, so can't speak to this difference... I'm curious about running it locally, but  the goal is to eventually incorporate the documentation linting into the editor itself so that authors can have a purely CMS interaction.